actors/builtin/power/power_test.go

```diff
@@ -22,10 +25,16 @@ import (
 func TestConstruction(t *testing.T) {
 	actor := spActorHarness{power.Actor{}, t}
 	powerActor := tutil.NewIDAddr(t, 100)
```

Comment:
Since you are already changing this file, can this be changed `builtin_spec.StoragePowerActorAddr`.